src/qc_ncsa.aplf

```diff
@@ -152,26 +152,52 @@
         H ← (÷2*0.5)×(2 2 ⍴ 1 1 1 ¯1)
 
         ⍝ P (phase shift)
-        P ← {2 2 ⍴ ((1 0 0),(*(0J1×⍵)))}
+        P ← {
+            ph ← *0J1×⍵
+            2 2 ⍴ ((1 0 0),ph)
+        }
 
         ⍝ T (4-th root of Z)
-        T ← 2 2 ⍴ ((1 0 0),(*(0J1×(0.25×○1))))
+        T ← 2 2 ⍴ ((1 0 0),*0J1×0.25×○1)
 
         ⍝ Rx (rotation in x)
-        Rx ← {2 2 ⍴ ((2○⍵÷2),(0J¯1×(1○⍵÷2)),(0J¯1×(1○⍵÷2)),(2○⍵÷2))}
+        Rx ← {
+            a←2○⍵÷2
+            b←0J¯1×1○⍵÷2
+            2 2 ⍴ a b b a
+        }
 
         ⍝ Ry (rotation in y)
-        Ry ← {2 2 ⍴ ((2○⍵÷2),(¯1×(1○⍵÷2)),(1○⍵÷2),(2○⍵÷2))}
+        Ry ← {
+            a ← 2○⍵÷2
+            b ← 1○⍵÷2
+            bm ← ¯1×b
+            2 2 ⍴ a bm b a
+        }
 
         ⍝ Rz (rotation in z)
-        Rz ← {2 2 ⍴ ((*¯0J1×(⍵÷2)),0,0,(*0J1×(⍵÷2)))}
+        Rz ← {
+            am ← *0J¯1×⍵÷2
+            ap ← *0J1×⍵÷2
+            2 2 ⍴ am 0 0 ap
+        }
```

Comment:
Same here, the return value changes from before:

![image](https://github.com/nunezco2/quAPL/assets/38033108/2a8ec687-645b-4ad4-9f7b-6e7167a69cc6)

Reply:
Idem.